### DISCUSSION TITLE
Concurrent update

### DISCUSSION THREAD

**Discussion by fghavam** [microsoft/go-mssqldb#223] [Q&A]
What is the best practice for handling concurrent updates of a row without encountering deadlock or exceeding the context timeout?

**Comment by fghavam** [comment]
any update on this question?

**Reply by dlevy-msft** [reply]
@fghavam - Can you share some more details on your scenario? The broader SQL language has quite a few patterns and practices that address concurrent updates. If you can share the problem you are running into, we can probably get you pointed in the right direction.
